sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/protocol/blob_rest_client.hpp

```diff
@@ -297,6 +297,7 @@ namespace Azure { namespace Storage { namespace Blobs {
 
     struct CreateAppendBlobResult
     {
+      bool Created = true;
```

Comment:
Isn't it better to expect `false` as the worst case (error case)?   
By using `true` we need to be more careful with this value during any exception

Reply:
I remember it was suggested to be `true` during arch board review meeting.

We set this value `false` on exception. So it's always correctly set, we don't have to worry about it.